packages/relay/src/lib/constants.ts

```diff
@@ -175,4 +175,10 @@ export default {
     NEW_HEADS: 'newHeads',
     NEW_PENDING_TRANSACTIONS: 'newPendingTransactions',
   },
+
+  // @source: Foundry related constants below can be found at https://github.com/Arachnid/deterministic-deployment-proxy?tab=readme-ov-file#latest-outputs
+  FOUNDRY_DETERMINISTIC_DEPLOYER_TRANSACTION:
```

Comment:
nit: Naming should be more general
```suggestion
  DETERMINISTIC_DEPLOYER_PROXY_TRANSACTION:
```

Reply:
Copy!